config/index.ts

```diff
@@ -57,14 +58,15 @@ const config = {
     'https://dev.cusconews.com',
   ],
   cacheRules: {
-    getAllUsers: 1000 * 60 * 60, // 1 hour . On User Created then we would need to clear it
+    allUsersData: 1000 * 60 * 60, // 1 hour . On User Created then we would need to clear it
+    userProfileData: 1000 * 60 * 60 * 2, // 2 hours
```

Comment:
should we also cache the feed data (posts) or do we want to keep it up-to-date?

Reply:
I will start soon with a new PR for cache improvements. 
We can then think about it.